calico.go

```diff
@@ -109,6 +109,17 @@ func cmdAdd(args *skel.CmdArgs) error {
 
 	var endpoint *api.WorkloadEndpoint
 
+	// If the prefix list returns 1 or more items, we go through the items and
+	// try to see if the name matches the WEP identifiers we have.
+	// The identifiers we use for this match at this point are:
+	// 1. Node name
+	// 2. Orchestrator ID ('cni' or 'k8s')
+	// 3. ContainerID
+	// 4. Pod name
+	// Note we don't use the interface name (endpoint) for this match.
+	// If we find a match from the returned list then we've found the workload endpoint,
+	// and we reuse that even if it has a different interface name, because
+	// we only support one interface per pod right now.
```

Comment:
Hmmm, the question is - with this comment, would you have understood the subtleties of whats going on?

We can't hold this up for this comment, but I was expecting to see in this comment, all those nitty gritty details that I went through with you today.

Reply:
Ok I've beefed up the comment with an example, PTAL